app/models/dataset_record/darwin_core/taxon.rb

```diff
@@ -286,6 +335,7 @@ def import(dwc_data_attributes = {})
         message: e.message,
         backtrace: e.backtrace
       }
+      raise if Rails.env.development?
```

Comment:
There is one line exactly like this one right below `rescue StandardError => e`. The purpose of re-raising is to crash in dev to inspect problems live rather than being recorded in record's metadata. This one must go away.